src/crewai/knowledge/source/base_file_knowledge_source.py

```diff
@@ -20,17 +23,42 @@ def model_post_init(self, _):
         self.content = self.load_content()
 
     def load_content(self) -> Dict[Path, str]:
-        """Load and preprocess file content. Should be overridden by subclasses."""
+        """Load and preprocess file content. Should be overridden by subclasses. We want to assume that the file path is relative to the project root in the knowledge directory."""
+
+        if isinstance(self.file_path, str):
+            self.file_path = self.convert_to_path(self.file_path)
+        elif isinstance(self.file_path, list):
+            processed_paths = []
+            for path in self.file_path:
+                processed_paths.append(self.convert_to_path(path))
+            self.file_path = processed_paths
+
         paths = [self.file_path] if isinstance(self.file_path, Path) else self.file_path
+        if not isinstance(paths, list):
+            raise ValueError("file_path must be a Path or a list of Paths")
 
         for path in paths:
             if not path.exists():
+                self._logger.log(
+                    "error",
+                    f"File not found: {path}. Try adding sources to the knowledge directory.",
+                    color="red",
+                )
                 raise FileNotFoundError(f"File not found: {path}")
             if not path.is_file():
+                self._logger.log(
+                    "error",
+                    f"Path is not a file: {path}",
+                    color="red",
+                )
                 raise ValueError(f"Path is not a file: {path}")
         return {}
 
     def save_documents(self, metadata: Dict[str, Any]):
         """Save the documents to the storage."""
         chunk_metadatas = [metadata.copy() for _ in self.chunks]
         self.storage.save(self.chunks, chunk_metadatas)
+
+    def convert_to_path(self, path: Union[Path, str]) -> Path:
+        """Convert a path to a Path object."""
+        return Path("knowledge/" + path) if isinstance(path, str) else path
```

Comment:
Should we change "knowledge/" to a constant to give our readers more context about what's going on.

Maybe add a `ROOT_KNOWLEDGE_DIR = "knowledge/"` at the top?